src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.cs

```diff
@@ -1577,15 +1573,16 @@ public bool Stretch
         /// </summary>
         [Browsable(false)]
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
+        [AllowNull]
         public ToolStripRenderer Renderer
         {
             get
             {
                 if (IsDropDown)
                 {
                     // PERF: since this is called a lot we don't want to make it virtual
-                    ToolStripDropDown dropDown = this as ToolStripDropDown;
-                    if (dropDown is ToolStripOverflow || dropDown.IsAutoGenerated)
+                    ToolStripDropDown? dropDown = this as ToolStripDropDown;
+                    if (dropDown is ToolStripOverflow || (dropDown is not null && dropDown.IsAutoGenerated))
```

Comment:
It is a dropDown and to keep the behavior unchanged,  I would change it with direct cast.
```suggestion
                    var dropDown = (ToolStripDropDown)this;
                    if (dropDown is ToolStripOverflow || dropDown.IsAutoGenerated)
```